yarn.lock

```diff
@@ -3670,14 +3670,14 @@ nise@^5.1.0:
     just-extend "^4.0.2"
     path-to-regexp "^1.7.0"
 
-njwt@^1.0.0:
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/njwt/-/njwt-1.1.0.tgz#9ae48b96df915dced5c2f49caebdd93c0950d708"
-  integrity sha512-lL9oQIc9GYy9ILyHpSTSEhcZHiB0yvCDBWf9EDGLYo2D+8oSZwzhIV5WV0bFEfIqmCIL720ZQyDCXyJM6YrpaQ==
+njwt@^2.0.1:
+  version "2.0.1"
+  resolved "https://registry.yarnpkg.com/njwt/-/njwt-2.0.1.tgz#21a224c33ab1468f903210b1f45a20181adb6600"
+  integrity sha512-HwFeZsPJ1aOhIjMjqT9Qv7BOsQbkxjRVPPSdFXNOTEkfKpr9+O6OX+dSN6TxxIErSYSqrmlDR4H2zOGOpEbZLA==
   dependencies:
     "@types/node" "^15.0.1"
     ecdsa-sig-formatter "^1.0.5"
-    uuid "^3.3.2"
+    uuid "^8.3.2"
```

Comment:
5 majors, no problems? It does seem like we already had it as a transitive dev dependency

```
=> Found "uuid@8.3.2"
info Has been hoisted to "uuid"
info Reasons this module exists
   - Hoisted from "njwt#uuid"
   - Hoisted from "jest-junit#uuid"
   - Hoisted from "@openapitools#openapi-generator-cli#@nestjs#common#uuid"
   - Hoisted from "@openapitools#openapi-generator-cli#@nestjs#core#uuid"
info Disk size without dependencies: "332KB"
info Disk size with unique dependencies: "332KB"
info Disk size with transitive dependencies: "332KB"
info Number of shared dependencies: 0
=> Found "istanbul-lib-processinfo#uuid@3.4.0"
info This module exists because "nyc#istanbul-lib-processinfo" depends on it.
info Disk size without dependencies: "92KB"
info Disk size with unique dependencies: "92KB"
info Disk size with transitive dependencies: "92KB"
info Number of shared dependencies: 0
```